test/modules/DirectMethodSender/Program.cs

```diff
@@ -37,14 +38,14 @@ public static async Task<int> MainAsync()
                 DateTime testStartAt = DateTime.UtcNow;
 
                 // BEARWASHERE: Create reportClient
```

Comment:
update comment to remove  "BEARWASHERE:"

Reply:
This is for my purpose of testing and will be removed before I turn this into an official PR.